tpchgen-cli/pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[build-system]
+requires = ["maturin>=1.0,<2.0"]
+build-backend = "maturin"
+
+[project]
+name = "tpchgen-cli"
+version = "0.1.0"
```

Comment:
not sure if we want to change to `1.0.0` to match rust

Reply:
I suggest we match the versions once we have confidence the workflow is running as expected